.travis.yml

```diff
@@ -0,0 +1,51 @@
+# We set the language to c because python isn't supported on the MacOS X nodes
+# on Travis. However, the language ends up being irrelevant anyway, since we
+# install Python ourselves using conda.
+language: c
+
+os:
+    - linux
+
+# Use Travis' container-based architecture
+sudo: false
+
+addons:
+  apt:
+    packages:
+    - graphviz
+    - texlive-latex-extra
+    - dvipng
+
+env:
+    global:
+        # The following versions are the 'default' for tests, unless
+        # overidden underneath. They are defined here in order to save having
+        # to repeat them for all configurations.
+        - PYTHON_VERSION=3.6
+        - PYTEST_VERSION=3.2
+        - PYTEST_COMMAND='pytest'
+        - NUMPY_VERSION=stable
+        - ASTROPY_VERSION=stable
+        - CONDA_DEPENDENCIES='six'
+
+    matrix:
```

Comment:
no need to have double matrix here, could move all these below, but doesn't matter much either.